tests/sles4sap/ipaddr2/configure.pm

```diff
@@ -35,8 +35,13 @@ sub run {
     record_info("TEST STAGE", "Prepare all the ssh connections within the 2 internal VMs");
     my $bastion_ip = ipaddr2_bastion_pubip();
     ipaddr2_bastion_key_accept(bastion_ip => $bastion_ip);
-    ipaddr2_internal_key_accept(bastion_ip => $bastion_ip);
-    ipaddr2_internal_key_gen(bastion_ip => $bastion_ip);
+
+    my %int_key_args = (bastion_ip => $bastion_ip);
+    # unsupported option "accept-new" for default ssh used in 12sp5
+    $int_key_args{key_checking} = 'no' if (check_var('IPADDR2_KEYCHECK_OLD', '1'));
```

Comment:
`check_var('IPADDR2_KEYCHECK_OLD', '1')` is working
`check_var('IPADDR2_KEYCHECK_OLD', 1)` does not

Reply:
Both should be the same: https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L744-L746

Do you have a link to where it didn't work. I'm curious about how it failed :)